controller.py

```diff
@@ -107,6 +170,8 @@ def canonicalize_author_name(self):
 class CatalogController(object):
     """A controller to manage a Collection's catalog"""
 
+    OPDS_CATALOG_REGISTRATION_MEDIA_TYPE = "application/opds+json;profile=https://librarysimplified.org/rel/profile/directory"
```

Comment:
Since we aren't serving links to OPDS servers, just an ID and the shared secret, you should be able to use the normal OPDS 2 media type.